### DISCUSSION TITLE
Cartoon Network [KR]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#899] [Q&A]
### Channel ID

CartoonNetwork.kr

### Notes (optional)

Sorry if i am making too many posts

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19868
